Code/Framework/AzNetworking/AzNetworking/UdpTransport/UdpNetworkInterface.h

```diff
@@ -55,8 +55,11 @@ namespace AzNetworking
     //! ### Fragmentation
     //! 
     //! If the raw packet size exceeds the configured maximum transmission unit (MTU) then the packet is broken into
-    //! multiple reliable fragments to avoid fragmentation at the routing level. Fragments are always reliable so the original
-    //! packet can be reconstructed. Operations that alter the payload generally follow this step so that they can be
+    //! multiple reliable fragments to avoid fragmentation at the routing level. Fragment reliability is controlled by the cvar
```

Comment:
Should that read, "...multiple fragments..." instead of "...multiple reliable fragments..."?

Reply:
Indeed it should, good eye!